### PR TITLE
OSD: display temperature and speed from ESC sensor.

### PR DESCRIPTION
**Pull request by rosek86**
Add option to display data from ESC sensor: temperature + speed.

### CODE REVIEW

File: src/main/io/osd.c [hunk 1]
```diff
@@ -587,6 +604,14 @@ void osdDrawElements(void)
         osdDrawSingleElement(OSD_GPS_LON);
     }
 #endif // GPS
+
+#ifdef USE_ESC_SENSOR
+  if (feature(FEATURE_ESC_SENSOR))
+  {
```
Comment (mikeller) [inline src/main/io/osd.c]: For consistency's sake, please use K & R style braces (brace on newline for functions, but on the same line for everything else.

Comment (rosek86) [inline src/main/io/osd.c]: thanks, this is corrected.

**Comment by mikeller** [comment]
@hydra: This should probably be rebased / merged after you've merged the OSD changes coming in from Betaflight.

**Comment by hydra** [comment]
done

**Comment by rosek86** [comment]
Thanks :-)

**Comment by mikeller** [comment]
@rosek86: Thanks for your contribution. Would it be possible that you added support for the new elements to the configurator as well?

**Comment by rosek86** [comment]
@mikeller Yep, I'm working on that, should be ready tonight.

**Pull request merged**